README.md

```diff
@@ -1,68 +1,26 @@
-# Google API Client [![Documentation](https://img.shields.io/badge/docs-Google%3A%3AApis-red.svg)](https://googleapis.dev/ruby/google-api-client/latest/Google/Apis.html) [![Gem Version](https://badge.fury.io/rb/google-api-client.svg)](https://badge.fury.io/rb/google-api-client)
+# Legacy REST Clients for Google APIs
 
-These client libraries are officially supported by Google.  However, the libraries are considered complete and are in maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
+This repository contains a set of simple client libraries for various Google APIs. These libraries are generated automatically from [Discovery Documents](https://developers.google.com/discovery), and the code generator is also hosted here in this repository.
 
-## Google Cloud Platform
+Each client provides:
 
-For Google Cloud Platform APIs such as Datastore, Cloud Storage or Pub/Sub, we recommend using [GoogleCloudPlatform/google-cloud-ruby](https://github.com/GoogleCloudPlatform/google-cloud-ruby) which is under active development.
+* A client object that connects to the HTTP/JSON REST endpoint for the service.
+* Ruby objects for data structures related to the service.
+* Integration with the googleauth gem for authentication using OAuth, API keys, and service accounts.
+* Control of retry, pagination, and timeouts.
 
-## Migrating from 0.8.x
+These client libraries are officially supported by Google, and are updated regularly to track changes to the service. However, they are considered legacy. A more modern client may be available for many Google services, especially Google Cloud Platform services such as Cloud Storage, Pub/Sub, and BigQuery. See the section below titled *Which client should I use?* for more information.
```

Comment:
Use an anchor link to *Which client should I use?*?

Reply:
Done